ocaml/alerts/certificate/certificate_check.ml

```diff
@@ -23,7 +25,7 @@ let generate_alert epoch (host, expiry) =
     let expired_message = "The TLS server certificate has expired." in
 
     let body msg =
-      Printf.sprintf "<message>%s</message><date>%s</date>" msg (Date.to_string expiry)
+      Printf.sprintf "<body><message>%s</message><date>%s</date></body>" msg (Date.to_string expiry)
```

Comment:
could `msg` ever come from some malicious user?

Reply:
No, the binding is local (in `general_alert`) and it's only fed local values (2 possibilities, `expiring_message` and `expired_mesage`)